lib/flame/pool.ex

```diff
@@ -721,25 +712,20 @@ defmodule FLAME.Pool do
 
     current_count = runner_count(state) + pending_count(state)
     new_count = strategy_module.desired_count(state, strategy_opts)
-
     num_tasks = max(new_count - current_count, 0)
 
-    if num_tasks do
-      tasks =
-        for _ <- 1..num_tasks do
-          Task.Supervisor.async_nolink(state.task_sup, fn ->
-            if on_grow_start, do: on_grow_start.(%{count: new_count, name: name, pid: self()})
-            start_child_runner(state)
-          end)
-        end
+    tasks =
+      for _ <- 1..num_tasks//1 do
```

Comment:
improved syntax by removing the check for `if num_tasks` since we can emulate it with the //1 flag on range

Reply:
nit: TBH I feel `if num_tasks do` is more readable here instead of doing an iteration from `1..0`

Reply:
the //1 syntax prevents doing the iteration from 1..0

Reply:
Sorry I meant the readability of it.